1_Lyalyushkin/AppList.py

```diff
@@ -0,0 +1,85 @@
+import ctypes
+import ctypes.util
+
+from CoreFoundation import *
```

Comment:
I would not recommend imports like `from something import *`.
It can confuse readers about the roots of some functions/classes. Explicit module/namespace qualification is always better.